djangoprojects/provision.py

```diff
@@ -21,25 +21,29 @@
     DEBUG=(bool, True)
 )
 
-# reading .env file
 environ.Env.read_env()
 
-api_token = env('API_KEY')
-api_url_base = 'https://api.digitalocean.com/v2/'
-headers = {'Content-Type': 'application/json', 'Authorization': 'Bearer {0}'.format(api_token)}
+API_TOKEN = env('API_KEY')
+API_URL_BASE = 'https://api.digitalocean.com/v2/'
+HEADERS = {'Content-Type': 'application/json', 'Authorization': 'Bearer {0}'.format(API_TOKEN)}
+
+with open(env('SSH_KEY_PATH'), 'r') as f:
+    ssh_key = f.readline()
+fingerprint = sshpubkeys.SSHKey(ssh_key)
 
 
 def list(droplet_name):
-    api_droplet = api_url_base + 'droplets?tag_name=' + droplet_name
-    response = requests.get(api_droplet, headers=headers)
-    jsondata = json.loads(response.text)
-    try:
-        if droplet_name == "Name:" + (jsondata.get("droplets")[0]).get("name") and response.status_code == 200 :
+    api_droplet = API_URL_BASE + 'droplets?tag_name=' + droplet_name
+    response = requests.get(api_droplet, headers=HEADERS)
+    jsondata = response.json()
+    droplets = jsondata['droplets']
+    if not len(droplets) == 0:
+        if droplet_name == droplets[0]['tags'][0] and response.status_code == 200 :
```

Comment:
Use `droplet_name in droplets[0]['tags']` because you may have more than one tag and position wont matter